pallets/farming/src/lib.rs

```diff
@@ -185,7 +185,7 @@ impl<T: Config> Pallet<T> {
             .try_into_balance()
             .unwrap_or(0);
 
-        if base_asset_amt < balance!(1) {
+        if base_asset_amt < Self::lp_min_xor_for_bonus_reward() {
```

Comment:
If the value < `lp_min_xor_for_bonus_reward` the 0 is returned as a weight. It means that if this value is too high, liquidity providers will not get any reward. @vovac12 is it intended?

Reply:
Yes, that's why we have this condition